.github/workflows/go-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: go-tests
+
+on: [push]
+
+env:
+  TEST_RESULTS: /tmp/test-results
+
+jobs:
+
+  go-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.19]
```

Comment:
Since this is a library that previously targeted go1.15.3, I wonder if we should add that version to the matrix as well to ensure we're still maintaining compatibility? If we don't want that, maybe we should explicitly call out this update in the PR description for the benefit of people looking back in the future?

Reply:
+1 -- excellent point, I'll add 1.15.3 into the matrix and also add a note that it was left there just in case.